gprofiler/profilers/perf.py

```diff
@@ -183,6 +183,7 @@ def __init__(
         self._metadata_collectors: List[PerfMetadata] = [GolangPerfMetadata(stop_event), NodePerfMetadata(stop_event)]
         self._insert_dso_name = insert_dso_name
         self._node_processes: List[Process] = []
+        self.pids_with_perf_maps: List[int] = []
```

Comment:
```suggestion
        self._node_processes_attached: List[int] = []
```

`_` to signify it's private
and the name to match the `_node_processes` attribute